nickhartdev/NewsHub
# Feature/startup screens

#### What's this PR do?  
- adds skeletons for the LanguageSelect and InterestSelect components
- adds unit tests for all components added
- adds PR template
#### Where should the reviewer start?
App.js
#### How should this be manually tested?
On starting the app, go to localhost:3000/language-select. You should see a list of languages pop up
#### Any background context you want to provide?
This is just a start at the initial component structure, styling will come later. I'm now shifting gears to hooking up i18next.
#### What are the relevant tickets?
Closes #5, closes #6
